handler/handler_test.go

```diff
@@ -59,13 +47,27 @@ var (
 	}
 )
 
-func TestContentPublishedHandler_Handle(t *testing.T) {
-	expectedSearchDataImport := marshalSearchDataImport(t, searchDataImportEvent)
+func TestHandlerForZebedeeReturingMandatoryFields(t *testing.T) {
```

Comment:
Typo - missing "n" in "Returing" => "Returning":
```suggestion
func TestHandlerForZebedeeReturningMandatoryFields(t *testing.T) {
```